app/client/src/components/shared/FullscreenContainer.tsx

```diff
@@ -0,0 +1,57 @@
+/** @jsxImportSource @emotion/react */
+
+import * as Dialog from '@radix-ui/react-dialog';
+import { WindowSize } from '@reach/window-size';
+// contexts
+import { useFullscreenState } from 'contexts/Fullscreen';
+// types
+import type { ReactNode } from 'react';
+
+type Props = {
+  children: ReactNode;
+  description?: string;
+  title?: string;
+};
+
+export default function FullscreenContainer({
+  children,
+  description,
+  title,
+}: Props) {
+  const { fullscreenActive, setFullscreenActive } = useFullscreenState();
+
+  return (
+    <WindowSize>
+      {({ width, height }) => (
+        <Dialog.Root open={fullscreenActive} onOpenChange={setFullscreenActive}>
+          <Dialog.Portal>
+            <Dialog.Content
+              css={{
+                backgroundColor: 'white',
+                height,
+                left: 0,
+                position: 'fixed',
+                top: 0,
+                width,
+                zIndex: 9999,
```

Comment:
```suggestion
                zIndex: 1000,
```

This is causing the glossary and modals to be hidden by the map when in full screen mode. `1000` is what the other modals are set to.

The waterbody popups have glossary definitions and info icons for testing modals.

Reply:
Good call, I completely missed checking those.